orderapi/src/main/java/fi/hel/verkkokauppa/order/api/data/accounting/CreateRefundAccountingRequestDto.java

```diff
@@ -0,0 +1,18 @@
+package fi.hel.verkkokauppa.order.api.data.accounting;
+
+import lombok.Data;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.List;
+
+@Data
+@Getter
```

Comment:
Getter and Setter are redundant because Data annotation provides both I think

Reply:
removed